internal/grid/muxserver.go

```diff
@@ -320,12 +321,9 @@ func (m *muxServer) close() {
 	m.cancel()
 	m.recvMu.Lock()
 	defer m.recvMu.Unlock()
-	if m.inbound != nil {
-		close(m.inbound)
-		m.inbound = nil
-	}
-	if m.outBlock != nil {
-		close(m.outBlock)
-		m.outBlock = nil
-	}
+	xioutil.SafeClose(m.inbound)
+	m.inbound = nil
+
+	xioutil.SafeClose(m.outBlock)
+	m.outBlock = nil
```

Comment:
Re-add the if - otherwise you will likely get excessive logging.

Reply:
done added back